google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java

```diff
@@ -146,4 +146,11 @@ static AdminClient create(AdminClientSettings settings) throws StatusException {
    *     status {@link io.grpc.Status.Code#NOT_FOUND}
    */
   ApiFuture<Void> deleteSubscription(SubscriptionPath path);
+
+  /**
+   * Tear down this admin client.
+   *
+   * @throws StatusException on a failure to properly terminate.
+   */
+  void close() throws StatusException;
```

Comment:
This is a breaking change that requires a major version update.

Reply:
There are no major versions in a pre-v1 library. This change will not be submitted as it is superceded by #295. This is also only a breaking change for someone who overrides (not uses) AdminClient, which is not an intended workflow. The previous method signature was `void close() throws Exception` which this is a subset of.